src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
+              if (!options.useContentExp) {
```

Comment:
I think I explained myself poorly. I'm not messing with the contentExp logic. I'm just moving the conditional inside the $observe because I also want to use the same $observe for watching the tooltip value and repositioning. I'm using one $observe for two different purposes instead of adding another $observe.